tests/integration/components/rental/image-test.js

```diff
@@ -7,6 +7,20 @@ module('Integration | Component | rental/image', function (hooks) {
   setupRenderingTest(hooks);
 
   test('it renders the given image', async function (assert) {
+    /*
+      These comments were moved from the HTML files because no comments are allowed in the hbs files
+      Once I feel like I know this stuff stronger I can delete these
+    */
+    /*
+      https://guides.emberjs.com/release/tutorial/part-1/more-about-components/#toc_forwarding-html-attributes-with-attributes
+      "... In general, it is a good idea to add ...attributes to the primary element 
+      in your component. This will allow for maximum flexibility, as the invoker 
+      may need to pass along classes for styling or ARIA attributes to improve 
+      accessibility."
+  
+      Note, below, the Rental::Image (double colon) syntax is the Namespaced Syntax
+      https://guides.emberjs.com/release/tutorial/part-1/more-about-components/#toc_organizing-code-with-namespaced-components 
+    */
```

Comment:
There is a rule preventing me from putting these in the HTML files themselves.

Seems like a good rule so I'm just going to keep it enabled and move the comments into the test files. Not a huge deal. Just a test app for my learning.

https://github.com/bgwest/ember-advanced-tutorial/pull/5#discussion_r1014492437